receiver/webhookeventreceiver/req_to_log.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhookeventreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/webhookeventreceiver"
+
+import (
+	"bufio"
+	"net/url"
+
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+func reqToLog(sc *bufio.Scanner,
+	query url.Values,
+	_config *Config,
+	idName string) (plog.Logs, int) {
+	log := plog.NewLogs()
+	resourceLog := log.ResourceLogs().AppendEmpty()
+	appendMetadata(resourceLog, query, idName)
+	scopeLog := resourceLog.ScopeLogs().AppendEmpty()
```

Comment:
If you want, and that can be a separate PR, you can add metadata to the Scope of the scopeLog.

Reply:
For sure! I think my approach to this is to iterate as it becomes more clear what type of information users of the tool expect to be shipped with their logs. I didn't want to add a bunch of stuff that wasn't important to anyone off the bat.